docs/plugins/development/creating-classic-plugins.asciidoc

```diff
@@ -0,0 +1,93 @@
+[[creating-classic-plugins]]
+=== Creating classic plugins
+
+Plugins provide {es} with mechanisms for custom authentication, authorization,
```

Comment:
Maybe use the term Class plugins here? "Plugins" isn't accurate, since there are the two types of plugins.